components/brave_shields/core/browser/ad_block_filters_provider.cc

```diff
@@ -45,6 +45,10 @@ void AdBlockFiltersProvider::NotifyObservers(bool is_for_default_engine) {
   }
 }
 
+bool AdBlockFiltersProvider::IsInitialized() {
+  return true;
```

Comment:
why is the default true instead of false?

Reply:
I guess this makes sense https://github.com/brave/brave-core/pull/22082/files#diff-4d045535d239d0bdab24eefe097d8f98e4d0d4d12c37714a0e3f34b387e94420R49